tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/GmcDetailsResource.java

```diff
@@ -91,7 +91,8 @@ public ResponseEntity<GmcDetailsDTO> updateGmcDetails(
       @RequestBody @Validated(Update.class) GmcDetailsDTO gmcDetailsDTO)
       throws URISyntaxException, MethodArgumentNotValidException {
     log.debug("REST request to update GmcDetails : {}", gmcDetailsDTO);
-    gmcDetailsValidator.validate(gmcDetailsDTO);
+    GmcDetailsDTO originalDto = gmcDetailsService.findOne(gmcDetailsDTO.getId());
+    gmcDetailsValidator.validate(gmcDetailsDTO, originalDto, Update.class);
```

Comment:
If refactoring, this could be placed after the id null check like `ContactDetailsResource.updateContactDetails()`.